### DISCUSSION TITLE
tutorials/client-side-graphql-react/02-graphql-basics

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#98] [course feedback]
# tutorials/client-side-graphql-react/02-graphql-basics

https://www.apollographql.com/tutorials/client-side-graphql-react/02-graphql-basics

**Comment by chilucdiep** [comment]
'folowed' is mistyped on this page (https://www.apollographql.com/tutorials/client-side-graphql-react/02-graphql-basics). Also writing this comment was difficult since the side menu goes on top of the comment section box

**Reply by mabuyo** [reply]
Thanks @chilucdiep - we'll fix that typo! Hmm, are you able to minimize the sidebar? There should be an icon at the top to do so, like this `<<`.

**Comment by kristiyan-velkov** [comment]
Amazing Tutorials! I really love them <3 

Thank you!

**Comment by backwardok** [comment]
Noticed that the UI changed a little bit since this tutorial was written, and the part where that's the most noticeable is with this line:
> From the Schema page, select the SDL option at the top. This will open up the entire GraphQL API—in its raw SDL—for you to browse!

Now this exists in the Schema menu under `Schema definition`. Would be great to update at least the instructions part, but also updating the screenshot would be helpful as well. Thanks!
